components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -97,16 +108,26 @@ function CojDeclarationSection() {
                 </SummaryList.Value>
               </SummaryList.Row>
             </SummaryList>
-            <Button
-              onClick={(e: { preventDefault: () => void }) => {
-                e.preventDefault();
-                handleSubmit();
-              }}
-              disabled={!isValid || isSubmitting}
-              data-cy="cogSignBtn"
-            >
-              Click to sign Conditions of Joining agreement
-            </Button>
+            {!canEdit ? (
+              <SummaryList noBorder>
+                <SummaryList.Row>
+                  <SummaryList.Value data-cy="SignedOn">
+                    Signed On: {DateUtilities.ToLocalDate(signedDate)}
+                  </SummaryList.Value>
+                </SummaryList.Row>
+              </SummaryList>
+            ) : (
+              <Button
+                onClick={(e: { preventDefault: () => void }) => {
+                  e.preventDefault();
+                  handleSubmit();
+                }}
+                disabled={!isValid || isSubmitting}
+                data-cy="cogSignBtn"
```

Comment:
May as well fix up this typo while you're making changes `cogSignBtn` -> `cojSignBtn`